repair_discount/models/mrp_repair.py

```diff
@@ -112,20 +68,20 @@ def _create_invoices(self, group=False):
                 fee_lines.invoice_line_id.with_context(
                     check_move_validity=False
                 ).update({"discount": fee_lines.discount})
-        self.invoice_id.with_context(
-            check_move_validity=False
-        )._recompute_dynamic_lines(
-            recompute_all_taxes=True, recompute_tax_base_amount=True
-        )
+
         return res
 
     def _calculate_line_base_price(self, line):
         return line.price_unit * (1 - (line.discount or 0.0) / 100.0)
 
+    amount_tax = fields.Monetary(
```

Comment:
Are you redefining this field as computed?

Reply:
Yes, you are right, this is not wrong